### PR TITLE
fix(curriculum): make logical AND step more concise

### LINKED ISSUE #49484: Basic JS steps need updates to facilitate translation
### Describe the Issue

Similar to issue #49408 
Please also update another step that has the same issue and should get a similar fix:
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator

The sentence:
"will only return Yes if num is greater than 5 and less than 10. The same logic can be written as:"
should be changed to:
"This will only..."
(adding This is enough to fix it as a reference to the previous code block)

And similarly for this step as well:
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-or-operator

### Affected Page

Multiple

### Your code

```

```

### Expected behavior

As described above

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by azr-arch**
removed grammatical mistakes stated in #49484

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #49484

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ if (num > 5) {
 return "No";
 ```
 
-will only return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written as:
+This will only return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written as:
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md:25]:
```suggestion
This code will return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written with the logical and operator.
```

Also, line 14 needs to be updated.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ if (num > 5) {
 return "No";
 ```
 
-will only return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written as:
```
[the remark this answers is not in the document]
Comment (azr-arch) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md:25]: how to attempt changes should i create new repo, if not how to do changes in a previous repo

[the remark this answers is not in the document]
Comment (azr-arch) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md:25]: btw i think  i`ve fixed it kindly check NVM above question

**Comment by azr-arch** [comment]
> 



> Hey @azr-arch
> 
> This PR seems to make similar changes as the existing PR <#49494>. As such, we are going to close this as duplicate.
> 
> If you feel you have additional changes to expand upon this PR, please feel free to push your commits and request this PR be reopened.
> 
> Thanks again! 😊
> 
> If you have any questions, feel free to ask questions on the ['Contributors' category on our forum](https://forum.freecodecamp.org/c/contributors) or [the contributors chat room](https://discord.gg/PRyKn3Vbay).

do they even get merged or they are just for educating newcomers?

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeremylt) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md:25]: This still does not use the requested wording.

**Comment by jeremylt** [comment]
Is this PR a different file than the other one (logical OR vs logical AND)? Really, one issue should  generally be one PR, but this is 2 PRs for one issue, which is confusing.

**Comment by Sboonny** [comment]
I prefer one PR, if it's two files.

It will be confusing for maintainers to see an PR for closed issue one, if one of them is merged.

**Comment by azr-arch** [comment]
> I prefer one PR, if it's two files.
> 
> It will be confusing for maintainers to see a PR for closed issue one, if one of them is merged.

i didnt know that i was supposed to fix all similar changes in one file sorry about that

**Comment by Sboonny** [comment]
> I prefer

This is subjective opinion, don't worry about it.

**Comment by azr-arch** [comment]
> 

i did commit suggestion does that fix the issue ? or should i have to make my own changes

**Comment by Sboonny** [comment]
Change line 14 as mentioned here https://github.com/freeCodeCamp/freeCodeCamp/pull/49495#discussion_r1117318012, then press Ready for review button

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeremylt) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md:25]: This still dose not use the requested wording

```
This code will return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written with the <defn>logical and</defn> operator.
```

**Comment by raisedadead** [comment]
Hi, 

We are closing old issues/pull requests that we think are stale. If you still need help or are fixing this, please open a new issue/pull request.

Thanks for your understanding. 

Happy contributing.

**Pull request closed without merging**